Modules/clinic/zipimport.c.h

```diff
@@ -32,7 +32,7 @@ zipimport_zipimporter___init__(PyObject *self, PyObject *args, PyObject *kwargs)
     int return_value = -1;
     PyObject *path;
 
-    if ((Py_TYPE(self) == &ZipImporter_Type) &&
+    if ((Py_TYPE(self) == &PyZipImporter_Type) &&
```

Comment:
@serhiy-storchaka I think I am having trouble updating the clinic stuff on this so it dont revert the changes to this file and build on travis properly.